client/devdocs/design/index.jsx

```diff
@@ -85,22 +85,27 @@ let DesignAssets = React.createClass( {
 	},
 
 	render() {
-		const { componentsUsageStats = {} } = this.props;
+		const { componentsUsageStats = {}, component } = this.props;
+		const { filter } = this.state;
+
 		return (
-			<div className="design-assets" role="main">
-				{
-					this.props.component
+			<div className="design" role="main">
```

Comment:
Applied and rebased. Both components and block pages are been updated using `<Main />` component. Also I've renamed and updated css names to `design`.